x/vm/wrappers/bank_test.go

```diff
@@ -525,20 +497,19 @@ func TestSendCoinsFromAccountToModule(t *testing.T) {
 			},
 		},
 		{
-			name:        "success - convert 18 decimals amount to 6 decimals with multiple tokens",
-			evmDenom:    TokenDenom,
-			evmDecimals: 6,
+			name:     "success - convert 18 decimals amount to 6 decimals with multiple tokens",
```

Comment:
Same as above, looks to be 6->18

Reply:
[ditto](https://github.com/cosmos/evm/pull/69#discussion_r2068352936)